Cargo.toml

```diff
@@ -15,15 +15,18 @@ license = "Apache-2.0"
 
 [workspace.dependencies]
 amplify = "4.6.0"
-bp-std = "0.11.0-beta.6"
-psbt = "0.11.0-beta.6"
-descriptors = "0.11.0-beta.6"
+# TODO: To test testnet4 upstream and downstream, 
+# git dependencies were used temporarily during development
+bp-std = { git = "https://github.com/will-bitlight/bp-std.git", branch = "testnet4"}
+psbt = { git = "https://github.com/will-bitlight/bp-std.git", branch = "testnet4"}
+descriptors = { git = "https://github.com/will-bitlight/bp-std.git", branch = "testnet4"}
```

Comment:
This has to go as a patch. It also has to use master branch since we have merged `testnet4` in `bp-std`